authenticity.cpp

```diff
@@ -98,7 +96,10 @@ std::string Authenticity::Assay
     // Read saved passkey from file.
     std::string passkey;
     {
-    fs::path passkey_path(data_path / "passkey");
+    fs::path passkey_path("passkey");
+    // Add only non-empty data path to avoid adding "./".
+    if(!data_path.empty())
+        passkey_path = data_path / passkey_path;
```

Comment:
I wonder if it's worth customizing our `operator/()` behaviour to handle first argument being empty specially.